src/data.rs

```diff
@@ -524,4 +605,63 @@ mod tests {
         let ord = Record::ordering(vec!["k1".to_string(), "k2".to_string()]);
         assert_eq!(ord(&r1, &r2), Ordering::Less);
     }
+
+    #[test]
+    fn arithmetic() {
+        assert_eq!(
+            Value::Float(OrderedFloat(5.)),
+            (Value::Float(OrderedFloat(4.)) + Value::Float(OrderedFloat(1.))).unwrap()
```

Comment:
if you like this (and a number of other tests can be cleaned up with the helper `Value::from_float`)